lib/tasks/jenkins.rake

```diff
@@ -1,25 +1,24 @@
 begin
   require "ci/reporter/rake/minitest"
 
-  namespace :foreman do
-    task :set_test_runner do
-      ENV['TESTOPTS'] = "#{ENV['TESTOPTS']} #{Rails.root}/test/test_runner.rb"
-    end
-  end
-
   namespace :jenkins do
-    task :unit => ["jenkins:setup:minitest", 'rake:test:units', 'rake:test:lib', 'rake:test:functionals']
-    task :integration => ["jenkins:setup:minitest", 'rake:test:integration']
-    task :lib => ["jenkins:setup:minitest", 'rake:test:lib']
-    task :functionals => ["jenkins:setup:minitest", 'rake:test:functionals']
-    task :units => ["jenkins:setup:minitest", 'rake:test:units']
+    task :unit => ['jenkins:db', 'jenkins:setup:minitest',
```

Comment:
It looks like just depending on `db:test:prepare` is sufficient, without needing our own rake task?